unittests/files/audioworklet/usage.ts

```diff
@@ -0,0 +1,8 @@
+registerProcessor(
```

Comment:
Added test in extra folder as other files has to be included. If other setup is needed say how to change it.